Gruntfile.js

```diff
@@ -126,5 +145,5 @@ module.exports = function(grunt) {
     grunt.registerTask('tests',       ['jasmine:dist:build']);
     grunt.registerTask('build',       ['jshint', 'clean', 'babel', 'addPolyfill', 'requirejs']);
     grunt.registerTask('default',     ['build']);
-
+    grunt.registerTask('sauce_tests', ['connect', 'saucelabs-jasmine']);
```

Comment:
To run sauce tests so they report to sauce labs, you'll need the username and API key as env variables (obviously not posting those here)